### ISSUE TITLE
Threading exceptions when using pywbem 1.7.0

### ISSUE THREAD

**Issue by andy-maier** [pywbem/pywbemtools#1383]
The test runs for pywbemcli when using pywbem 1.7.0 in PR https://github.com/pywbem/pywbemtools/pull/1382 fail because there is an unexpected message about an ignored threading exception.

From https://github.com/pywbem/pywbemtools/actions/runs/8702665593/job/23867213304?pr=1382#step:20:127

```
E       AssertionError: Unexpected number of lines on stderr in test:
E       Verify failure of 'start' with invalid bind host name and http,
E       Expected patterns (cnt=1):
E       ------------
E       Cannot start listener .+: .*
E       ------------
E       Actual lines (cnt=8):
E       ------------
E       Error: Cannot start listener lis1: [Errno -3] Temporary failure in name resolution
E       Exception ignored in: <module 'threading' from '/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/threading.py'>
E       Traceback (most recent call last):
E         File "/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/threading.py", line 1567, in _shutdown
E           lock.acquire()
E         File "/home/runner/work/pywbemtools/pywbemtools/pywbemtools/pywbemlistener/_cmd_listener.py", line 932, in run_term_signal_handler
E           raise SystemExit(1)
E       SystemExit: 1
E       ------------
```

**Comment by KSchopmeyer** [comment]
This issue is caused by code introduces in pywbem 1.8.0 and starting a listener where the start fails with an exception.  That causes the pywbemlistener listener process to hang because the callback queue does not exit.  See pywbem issue 
 pywbem issue https://github.com/pywbem/pywbem/issues/3157 which was closed with the release of pywbem 1.7.1.  

Therefore I propose closing this issue with no changes to pywbemtools.

**Comment by KSchopmeyer** [comment]
Close as fixed by release of pywbem 1.7.1
